edit-game/index.md

```diff
@@ -0,0 +1,15 @@
+## How do I edit/manage my game?
+
+Head over to your [dashboard](http://gamejolt.com/dashboard/) and select the game from the list under "Your Games". You will be taken to the game's management page. Here's where you'll manage and edit every aspect of your game.
+
+You begin on the Overview tab. If you click the icon next to "Overview", you'll see a menu of your options, including "Edit Game", "Thumbnail", "Page Header", and "Maturity Rating". 
```

Comment:
"If you click the icon next to "Overview" " not sure what icon this is referring to. Can we clarify? "Edit Game" is now "Edit Details" can we update?